composer.json

```
@@ -21,10 +21,11 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
-        "symfony/framework-bundle": "~3.4|~4.0|~5.0",
+        "symfony/framework-bundle": "~3.4|~4.0|~5.0|6.0.x@dev",
```

```suggestion
        "symfony/framework-bundle": "~3.4|~4.0|~5.0|~6.0",
```

This is redundant if you lower the minimum stability globally. 😉